lib/y2lan_restart_common.pm

```diff
@@ -407,9 +408,16 @@ If network is controlled by Networkmanager, don't change any network settings.
 =cut
 
 sub open_yast2_lan {
-    my $is_nm = !script_run('systemctl is-active NetworkManager');    # Revert boolean because of bash vs perl's return code.
-
-    $module_name = y2_module_consoletest::yast2_console_exec(yast2_module => 'lan');
+    my %options = @_;
```

Comment:
Why need the parameter for open_yast2_lan? Just judge the PowerVM inside the function and set correct $y2_opts is ok.

Reply:
I suggest to add some comments about how to use this function because of the new argument, this make users clear.

Reply:
1.open_yast2_lan already used in many place in current code, so i prefer extend parameter have ability to handle future change.

2.Comments already added for new argument.